From: Anonymous <noreply@weblate.org>
Date: Tue, 25 Aug 2026 07:43:34 +0000
Subject: =?utf-8?b?VHJhbnNsYXRlZCB1c2luZyBXZWJsYXRlICjmnJ3pspzor60gKGtvX0tQ?=
 =?utf-8?b?KSk=?=

Currently translated at 100.0% (33 of 33 strings)

Translation: openkylin-huanghe/kylin-file-crush
Translate-URL: http://weblate.openkylin.top/projects/openkylin-huanghe/kylin-file-crush/ko_KP/
---
 .../kylin-os-manager-session-service_de.ts         | 168 +++++
 .../translations/fault-diagnosis_de.ts             | 773 +++++++++++++++++++++
 plugins/junk_clean/translations/junk-clean_de.ts   | 442 ++++++++++++
 .../translations/kylin-file-crush_ko_KP.ts         |  66 +-
 4 files changed, 1416 insertions(+), 33 deletions(-)
 create mode 100644 daemons/sessionservice/translations/kylin-os-manager-session-service_de.ts
 create mode 100644 plugins/fault_diagnosis/translations/fault-diagnosis_de.ts
 create mode 100644 plugins/junk_clean/translations/junk-clean_de.ts

diff --git a/daemons/sessionservice/translations/kylin-os-manager-session-service_de.ts b/daemons/sessionservice/translations/kylin-os-manager-session-service_de.ts
new file mode 100644
index 0000000..54bd738
--- /dev/null
+++ b/daemons/sessionservice/translations/kylin-os-manager-session-service_de.ts
@@ -0,0 +1,168 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="de">
+<context>
+    <name>DiskMonitorExceptionPopup</name>
+    <message>
+        <location filename="../disk_monitor_exception_popup.cpp" line="25"/>
+        <source>Kylin Os Manager</source>
+        <translation>Kylin Manager</translation>
+    </message>
+    <message>
+        <location filename="../disk_monitor_exception_popup.cpp" line="54"/>
+        <source>Not prompted within 3 days</source>
+        <translation>3 Tage lang nicht mehr anzeigen</translation>
+    </message>
+    <message>
+        <location filename="../disk_monitor_exception_popup.cpp" line="66"/>
+        <source>Cancel</source>
+        <translation>Abbrechen</translation>
+    </message>
+    <message>
+        <location filename="../disk_monitor_exception_popup.cpp" line="73"/>
+        <source>Clean up immediately</source>
+        <translation>Jetzt bereinigen</translation>
+    </message>
+</context>
+<context>
+    <name>QObject</name>
+    <message>
+        <location filename="../driverinstall/aptinstaller.cpp" line="68"/>
+        <source>Install timeout.</source>
+        <translation>Installation abgelaufen.</translation>
+    </message>
+</context>
+<context>
+    <name>ServiceSupportPopup</name>
+    <message>
+        <location filename="../service_support_popup.cpp" line="22"/>
+        <source>Kylin Os Manager</source>
+        <translation>Kylin Manager</translation>
+    </message>
+    <message>
+        <location filename="../service_support_popup.cpp" line="40"/>
+        <source>Detected system abnormality, please contact the technical service immediately or call 400!</source>
+        <translation>Systemanomalie erkannt, bitte kontaktieren Sie sofort den technischen Support oder rufen Sie die 400-Hotline an!</translation>
+    </message>
+    <message>
+        <location filename="../service_support_popup.cpp" line="57"/>
+        <source>Or contact us</source>
+        <translation>oder kontaktieren Sie uns</translation>
+    </message>
+    <message>
+        <location filename="../service_support_popup.cpp" line="61"/>
+        <source>Telephone</source>
+        <translation>Mobiltelefon</translation>
+    </message>
+    <message>
+        <location filename="../service_support_popup.cpp" line="67"/>
+        <source>Mail</source>
+        <translation>E-Mail</translation>
+    </message>
+    <message>
+        <location filename="../service_support_popup.cpp" line="89"/>
+        <source>For common questions, please refer to</source>
+        <translation>Für häufig gestellte Fragen siehe</translation>
+    </message>
+    <message>
+        <location filename="../service_support_popup.cpp" line="92"/>
+        <source>KylinOs offical website</source>
+        <translation>Offizielle Website von Kylin Software</translation>
+    </message>
+    <message>
+        <location filename="../service_support_popup.cpp" line="133"/>
+        <source>Cancel</source>
+        <translation>Abbrechen</translation>
+    </message>
+    <message>
+        <location filename="../service_support_popup.cpp" line="140"/>
+        <source>Clean</source>
+        <translation>Bereinigen</translation>
+    </message>
+</context>
+<context>
+    <name>SessionClient</name>
+    <message>
+        <location filename="../sessionclient.cpp" line="126"/>
+        <location filename="../sessionclient.cpp" line="175"/>
+        <location filename="../sessionclient.cpp" line="221"/>
+        <location filename="../sessionclient.cpp" line="275"/>
+        <location filename="../sessionclient.cpp" line="357"/>
+        <location filename="../sessionclient.cpp" line="360"/>
+        <location filename="../sessionclient.cpp" line="470"/>
+        <location filename="../sessionclient.cpp" line="473"/>
+        <source>Kylin Os Manager</source>
+        <translation>Kylin Manager</translation>
+    </message>
+    <message>
+        <location filename="../sessionclient.cpp" line="129"/>
+        <location filename="../sessionclient.cpp" line="178"/>
+        <location filename="../sessionclient.cpp" line="224"/>
+        <location filename="../sessionclient.cpp" line="278"/>
+        <source>Device Manager</source>
+        <translation>Geräteverwaltung</translation>
+    </message>
+    <message>
+        <location filename="../sessionclient.cpp" line="130"/>
+        <source>The system detects that a new device has an installable driver. Do you want to install?</source>
+        <translation>Neues installierbares Treibergerät erkannt. Treiber installieren?</translation>
+    </message>
+    <message>
+        <location filename="../sessionclient.cpp" line="131"/>
+        <source>Close</source>
+        <translation>Abbrechen</translation>
+    </message>
+    <message>
+        <location filename="../sessionclient.cpp" line="131"/>
+        <source>Install</source>
+        <translation>Installieren</translation>
+    </message>
+    <message>
+        <location filename="../sessionclient.cpp" line="179"/>
+        <source>Start installing the corresponding driver</source>
+        <translation>Installation des entsprechenden Treibers starten</translation>
+    </message>
+    <message>
+        <location filename="../sessionclient.cpp" line="225"/>
+        <source>Driver Installed</source>
+        <translation>Treiber installiert</translation>
+    </message>
+    <message>
+        <location filename="../sessionclient.cpp" line="227"/>
+        <source>Open Scanner</source>
+        <translation>Scan öffnen</translation>
+    </message>
+    <message>
+        <location filename="../sessionclient.cpp" line="279"/>
+        <source>Driver install failed.</source>
+        <translation>Treiberinstallation fehlgeschlagen.</translation>
+    </message>
+    <message>
+        <location filename="../sessionclient.cpp" line="361"/>
+        <source>Detected log files that are too large, it is recommended to clean them up</source>
+        <translation>Die Protokolldatei ist zu groß. Eine Bereinigung wird empfohlen.</translation>
+    </message>
+    <message>
+        <location filename="../sessionclient.cpp" line="364"/>
+        <location filename="../sessionclient.cpp" line="477"/>
+        <source>Cancel</source>
+        <translation>Abbrechen</translation>
+    </message>
+    <message>
+        <location filename="../sessionclient.cpp" line="366"/>
+        <location filename="../sessionclient.cpp" line="479"/>
+        <source>One click cleaning</source>
+        <translation>Ein-Klick-Bereinigung</translation>
+    </message>
+    <message>
+        <location filename="../sessionclient.cpp" line="398"/>
+        <source>Detected log files that are too large, please go to Kylin Os Manager for cleaning!</source>
+        <translation>Die Protokolldatei ist zu groß. Bitte bereinigen Sie sie im Kylin Manager!</translation>
+    </message>
+    <message>
+        <location filename="../sessionclient.cpp" line="474"/>
+        <source>The avaliable disk space is less than %1%, please clean it up in a timely manner</source>
+        <translation>Freier Speicherplatz auf der Festplatte unter %1 %. Bitte bereinigen Sie rechtzeitig.</translation>
+    </message>
+</context>
+</TS>
diff --git a/plugins/fault_diagnosis/translations/fault-diagnosis_de.ts b/plugins/fault_diagnosis/translations/fault-diagnosis_de.ts
new file mode 100644
index 0000000..2b090d3
--- /dev/null
+++ b/plugins/fault_diagnosis/translations/fault-diagnosis_de.ts
@@ -0,0 +1,773 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="de">
+<context>
+    <name>ConfigWin</name>
+    <message>
+        <location filename="../3rdparty/internal_network_diagnosis/config_win.ui" line="32"/>
+        <source>Form</source>
+        <translation></translation>
+    </message>
+    <message>
+        <location filename="../3rdparty/internal_network_diagnosis/config_win.ui" line="264"/>
+        <source>SetInner</source>
+        <translation>Intranet-Erkennung aktivieren</translation>
+    </message>
+    <message>
+        <location filename="../3rdparty/internal_network_diagnosis/config_win.ui" line="429"/>
+        <location filename="../3rdparty/internal_network_diagnosis/config_win.cpp" line="197"/>
+        <source>Cancel</source>
+        <translation>Abbrechen</translation>
+    </message>
+    <message>
+        <location filename="../3rdparty/internal_network_diagnosis/config_win.ui" line="464"/>
+        <location filename="../3rdparty/internal_network_diagnosis/config_win.cpp" line="198"/>
+        <source>Save</source>
+        <translation>Speichern</translation>
+    </message>
+    <message>
+        <location filename="../3rdparty/internal_network_diagnosis/config_win.cpp" line="159"/>
+        <location filename="../3rdparty/internal_network_diagnosis/config_win.cpp" line="172"/>
+        <source>IntraNetSet</source>
+        <translation>Intranet-Diagnoseeinstellungen</translation>
+    </message>
+</context>
+<context>
+    <name>IPWebWidget</name>
+    <message>
+        <location filename="../3rdparty/internal_network_diagnosis/ipweb_widget.ui" line="32"/>
+        <source>Form</source>
+        <translation></translation>
+    </message>
+    <message>
+        <location filename="../3rdparty/internal_network_diagnosis/ipweb_widget.ui" line="93"/>
+        <source>Addr</source>
+        <translation>Adresse</translation>
+    </message>
+    <message>
+        <location filename="../3rdparty/internal_network_diagnosis/ipweb_widget.cpp" line="55"/>
+        <source>IP</source>
+        <translation>IP-Adresse</translation>
+    </message>
+    <message>
+        <location filename="../3rdparty/internal_network_diagnosis/ipweb_widget.cpp" line="57"/>
+        <source>Website</source>
+        <translation>Webadresse</translation>
+    </message>
+    <message>
+        <location filename="../3rdparty/internal_network_diagnosis/ipweb_widget.cpp" line="101"/>
+        <source>Format error,IP is invalid</source>
+        <translation>Fehlerhaftes Format, IP-Adresse ungültig</translation>
+    </message>
+    <message>
+        <location filename="../3rdparty/internal_network_diagnosis/ipweb_widget.cpp" line="114"/>
+        <source>Format error,web is invalid</source>
+        <translation>Ungültiges Format, Webadresse ungültig</translation>
+    </message>
+</context>
+<context>
+    <name>IncreaseWidget</name>
+    <message>
+        <location filename="../3rdparty/internal_network_diagnosis/increase_widget.ui" line="32"/>
+        <source>Form</source>
+        <translation></translation>
+    </message>
+</context>
+<context>
+    <name>K::UserExperienceDialog</name>
+    <message>
+        <location filename="../component/user_experience_dialog.cpp" line="38"/>
+        <source>Have all issues been fixed?</source>
+        <translation>Wurden alle Probleme behoben?</translation>
+    </message>
+    <message>
+        <location filename="../component/user_experience_dialog.cpp" line="54"/>
+        <source>If there are any unresolved issues, you can click &quot;Submit feedback&quot; to let us know about the problem.</source>
+        <translation>Falls nicht behobene Probleme vorhanden sind, können Sie auf &quot;Feedback senden&quot; klicken, um uns über das Problem zu informieren.</translation>
+    </message>
+    <message>
+        <location filename="../component/user_experience_dialog.cpp" line="62"/>
+        <source>Submit feedback</source>
+        <translation>Feedback senden</translation>
+    </message>
+    <message>
+        <location filename="../component/user_experience_dialog.cpp" line="66"/>
+        <source>Fixed</source>
+        <translation>Behoben</translation>
+    </message>
+</context>
+<context>
+    <name>MenuModule</name>
+    <message>
+        <location filename="../3rdparty/internal_network_diagnosis/menumodule.cpp" line="66"/>
+        <location filename="../3rdparty/internal_network_diagnosis/menumodule.cpp" line="102"/>
+        <source>Help</source>
+        <translation>Hilfe</translation>
+    </message>
+    <message>
+        <location filename="../3rdparty/internal_network_diagnosis/menumodule.cpp" line="68"/>
+        <location filename="../3rdparty/internal_network_diagnosis/menumodule.cpp" line="100"/>
+        <source>About</source>
+        <translation>Über</translation>
+    </message>
+    <message>
+        <location filename="../3rdparty/internal_network_diagnosis/menumodule.cpp" line="70"/>
+        <location filename="../3rdparty/internal_network_diagnosis/menumodule.cpp" line="105"/>
+        <source>Configure</source>
+        <translation>Einstellungen</translation>
+    </message>
+    <message>
+        <location filename="../3rdparty/internal_network_diagnosis/menumodule.cpp" line="72"/>
+        <location filename="../3rdparty/internal_network_diagnosis/menumodule.cpp" line="98"/>
+        <source>Quit</source>
+        <translation>Beenden</translation>
+    </message>
+    <message>
+        <location filename="../3rdparty/internal_network_diagnosis/menumodule.cpp" line="134"/>
+        <location filename="../3rdparty/internal_network_diagnosis/menumodule.cpp" line="253"/>
+        <source>Version: </source>
+        <translation>Version: </translation>
+    </message>
+    <message>
+        <location filename="../3rdparty/internal_network_diagnosis/menumodule.cpp" line="135"/>
+        <source>Network-check-tool is a software that can quickly detect,diagnose,and optimize networks.</source>
+        <translation>Das Netzwerkdiagnosetool ist eine Software zur schnellen Überprüfung, Diagnose und Optimierung des Netzwerks mit einem einfachen und benutzerfreundlichen Bedienungsablauf.</translation>
+    </message>
+    <message>
+        <location filename="../3rdparty/internal_network_diagnosis/menumodule.cpp" line="257"/>
+        <source>Network-check-tool is a software that can quickly detect, diagnose, and optimize networks. </source>
+        <translation>Das Netzwerkdiagnosetool ist eine Software zur schnellen Erkennung, Diagnose und Optimierung des Netzwerks mit einem einfachen und bequemen Bedienungsablauf. </translation>
+    </message>
+    <message>
+        <location filename="../3rdparty/internal_network_diagnosis/menumodule.cpp" line="269"/>
+        <location filename="../3rdparty/internal_network_diagnosis/menumodule.cpp" line="352"/>
+        <location filename="../3rdparty/internal_network_diagnosis/menumodule.cpp" line="360"/>
+        <source>Service &amp; Support: </source>
+        <translation>Service- und Support-Team: </translation>
+    </message>
+</context>
+<context>
+    <name>QObject</name>
+    <message>
+        <location filename="../fault_diagnosis.cpp" line="106"/>
+        <source>Health examination</source>
+        <translation>Gesundheitsprüfung</translation>
+    </message>
+    <message>
+        <location filename="../fault_diagnosis.cpp" line="108"/>
+        <source>FaultDiagnosis</source>
+        <translation>Fehlerdiagnose</translation>
+    </message>
+</context>
+<context>
+    <name>QuadBtnsTitleBar</name>
+    <message>
+        <location filename="../3rdparty/internal_network_diagnosis/quad_btns_title_bar.cpp" line="30"/>
+        <source>menu</source>
+        <translation>Optionen</translation>
+    </message>
+    <message>
+        <location filename="../3rdparty/internal_network_diagnosis/quad_btns_title_bar.cpp" line="38"/>
+        <source>minimize</source>
+        <translation>Minimieren</translation>
+    </message>
+    <message>
+        <location filename="../3rdparty/internal_network_diagnosis/quad_btns_title_bar.cpp" line="51"/>
+        <source>full screen</source>
+        <translation>Maximieren</translation>
+    </message>
+    <message>
+        <location filename="../3rdparty/internal_network_diagnosis/quad_btns_title_bar.cpp" line="64"/>
+        <source>close</source>
+        <translation>Schließen</translation>
+    </message>
+</context>
+<context>
+    <name>fault_diagnosis::DiagnosisCore</name>
+    <message>
+        <location filename="../diagnosis_core.cpp" line="288"/>
+        <source>Repair fail</source>
+        <translation>Reparatur fehlgeschlagen</translation>
+    </message>
+</context>
+<context>
+    <name>fault_diagnosis::DiagnosisEntryWidget</name>
+    <message>
+        <location filename="../diagnosis_entry_widget.cpp" line="39"/>
+        <source>Please contact technical service</source>
+        <translation>Bitte kontaktieren Sie den technischen Support</translation>
+    </message>
+    <message>
+        <location filename="../diagnosis_entry_widget.cpp" line="42"/>
+        <source>Go clean up</source>
+        <translation>Bereinigen</translation>
+    </message>
+    <message>
+        <location filename="../diagnosis_entry_widget.cpp" line="107"/>
+        <source>Normal</source>
+        <translation>Normal</translation>
+    </message>
+    <message>
+        <location filename="../diagnosis_entry_widget.cpp" line="112"/>
+        <source>Exception</source>
+        <translation>Anomalie</translation>
+    </message>
+    <message>
+        <location filename="../diagnosis_entry_widget.cpp" line="117"/>
+        <source>Diagnosising...</source>
+        <translation>Überprüfung läuft...</translation>
+    </message>
+    <message>
+        <location filename="../diagnosis_entry_widget.cpp" line="122"/>
+        <source>Fixed</source>
+        <translation>Repariert</translation>
+    </message>
+    <message>
+        <location filename="../diagnosis_entry_widget.cpp" line="127"/>
+        <source>Repairing...</source>
+        <translation>Reparatur läuft...</translation>
+    </message>
+    <message>
+        <location filename="../diagnosis_entry_widget.cpp" line="132"/>
+        <source>Repair failed</source>
+        <translation>Reparatur fehlgeschlagen</translation>
+    </message>
+</context>
+<context>
+    <name>fault_diagnosis::DiagnosisGroupWidget</name>
+    <message>
+        <location filename="../diagnosis_group_widget.cpp" line="63"/>
+        <source>Scanned %1 items</source>
+        <translation>%1 Elemente gescannt</translation>
+    </message>
+</context>
+<context>
+    <name>fault_diagnosis::DiagnosisSubentryWidget</name>
+    <message>
+        <location filename="../diagnosis_subentry_widget.cpp" line="37"/>
+        <source>Please contact technical service</source>
+        <translation>Bitte kontaktieren Sie den technischen Support</translation>
+    </message>
+    <message>
+        <location filename="../diagnosis_subentry_widget.cpp" line="41"/>
+        <source>Go clean up</source>
+        <translation>Bereinigen</translation>
+    </message>
+    <message>
+        <location filename="../diagnosis_subentry_widget.cpp" line="80"/>
+        <source>Diagnosising...</source>
+        <translation>Überprüfung läuft...</translation>
+    </message>
+    <message>
+        <location filename="../diagnosis_subentry_widget.cpp" line="85"/>
+        <source>Normal</source>
+        <translation>Normal</translation>
+    </message>
+    <message>
+        <location filename="../diagnosis_subentry_widget.cpp" line="90"/>
+        <source>Exception</source>
+        <translation>Anomalie</translation>
+    </message>
+    <message>
+        <location filename="../diagnosis_subentry_widget.cpp" line="95"/>
+        <source>Repairing...</source>
+        <translation>Wird repariert...</translation>
+    </message>
+    <message>
+        <location filename="../diagnosis_subentry_widget.cpp" line="100"/>
+        <source>Fixed</source>
+        <translation>Repariert</translation>
+    </message>
+    <message>
+        <location filename="../diagnosis_subentry_widget.cpp" line="105"/>
+        <source>Repair failed</source>
+        <translation>Reparatur fehlgeschlagen</translation>
+    </message>
+</context>
+<context>
+    <name>fault_diagnosis::ErrorEntryWidget</name>
+    <message>
+        <location filename="../error_entry_widget.cpp" line="32"/>
+        <source>Manual repair</source>
+        <translation>Manuelle Reparatur</translation>
+    </message>
+    <message>
+        <location filename="../error_entry_widget.cpp" line="122"/>
+        <source>Exception</source>
+        <translation>Anomalie</translation>
+    </message>
+    <message>
+        <location filename="../error_entry_widget.cpp" line="127"/>
+        <source>Repairing...</source>
+        <translation>Wird repariert...</translation>
+    </message>
+    <message>
+        <location filename="../error_entry_widget.cpp" line="132"/>
+        <source>Fixed</source>
+        <translation>Behoben</translation>
+    </message>
+    <message>
+        <location filename="../error_entry_widget.cpp" line="199"/>
+        <source>We currently do not support fixing this issue. Stay tuned!</source>
+        <translation>Die Reparatur dieses Problems wird derzeit nicht unterstützt, bitte haben Sie etwas Geduld!</translation>
+    </message>
+    <message>
+        <location filename="../error_entry_widget.cpp" line="207"/>
+        <source>Fail to create solution.</source>
+        <translation>Fehler beim Generieren der Lösung.</translation>
+    </message>
+</context>
+<context>
+    <name>fault_diagnosis::FaultDiagnosis</name>
+    <message>
+        <location filename="../fault_diagnosis.cpp" line="162"/>
+        <source>Fault Diagnosis</source>
+        <translation>Fehlerdiagnose</translation>
+    </message>
+    <message>
+        <location filename="../fault_diagnosis.cpp" line="162"/>
+        <source>Currently running, please try again after finishing.</source>
+        <translation>Derzeit in Betrieb, bitte versuchen Sie es nach Abschluss erneut.</translation>
+    </message>
+    <message>
+        <location filename="../fault_diagnosis.cpp" line="218"/>
+        <source>The module is not supported</source>
+        <translation>Reparaturkomponente nicht unterstützt</translation>
+    </message>
+    <message>
+        <location filename="../fault_diagnosis.cpp" line="234"/>
+        <source>The diagnostic type is not supported</source>
+        <translation>Überprüfungstyp wird nicht unterstützt</translation>
+    </message>
+</context>
+<context>
+    <name>fault_diagnosis::HomePageWidget</name>
+    <message>
+        <location filename="../home_page_widget.cpp" line="99"/>
+        <source>One click troubleshooting of computer malfunctions</source>
+        <translation>PC-Probleme mit einem Klick diagnostizieren</translation>
+    </message>
+    <message>
+        <location filename="../home_page_widget.cpp" line="101"/>
+        <source>Quick detection and repair of fault issues, safeguarding the health of your computer</source>
+        <translation>Schnelle Erkennung und Reparatur von Fehlern, für die Gesundheit Ihres Computers</translation>
+    </message>
+    <message>
+        <location filename="../home_page_widget.cpp" line="181"/>
+        <source>Network detection</source>
+        <translation>Netzwerkdiagnose</translation>
+    </message>
+    <message>
+        <location filename="../home_page_widget.cpp" line="108"/>
+        <location filename="../home_page_widget.cpp" line="318"/>
+        <source>One click detection</source>
+        <translation>Ein-Klick-Diagnose</translation>
+    </message>
+    <message>
+        <location filename="../home_page_widget.cpp" line="171"/>
+        <source>Special cleaning to free up space</source>
+        <translation>Spezielle Bereinigung, Speicherplatz freigeben</translation>
+    </message>
+    <message>
+        <location filename="../home_page_widget.cpp" line="182"/>
+        <source>Troubleshooting issues with inablilty to access the internet</source>
+        <translation>Probleme bei der Internetverbindung beheben</translation>
+    </message>
+    <message>
+        <location filename="../home_page_widget.cpp" line="187"/>
+        <source>System components and performance detection</source>
+        <translation>Systemkomponenten und Leistungsprüfung</translation>
+    </message>
+    <message>
+        <location filename="../home_page_widget.cpp" line="188"/>
+        <source>Resolve hardware health and system performance issues</source>
+        <translation>Behebung von Hardware-Gesundheits- und Systemleistungsproblemen</translation>
+    </message>
+    <message>
+        <location filename="../home_page_widget.cpp" line="193"/>
+        <source>Application detection</source>
+        <translation>Anwendungserkennung</translation>
+    </message>
+    <message>
+        <location filename="../home_page_widget.cpp" line="194"/>
+        <source>Application caching and permission settings</source>
+        <translation>Anwendungscache und Berechtigungseinstellungen</translation>
+    </message>
+    <message>
+        <location filename="../home_page_widget.cpp" line="200"/>
+        <source>System update detection</source>
+        <translation>Systemaktualisierungsprüfung</translation>
+    </message>
+    <message>
+        <location filename="../home_page_widget.cpp" line="202"/>
+        <source>Online system update detection</source>
+        <translation>Online-Systemaktualisierungsprüfung</translation>
+    </message>
+    <message>
+        <location filename="../home_page_widget.cpp" line="204"/>
+        <source>Resolve issues encountered during updates and upgrades</source>
+        <translation>Probleme bei Updates und Upgrades beheben</translation>
+    </message>
+    <message>
+        <location filename="../home_page_widget.cpp" line="209"/>
+        <source>Software store detection</source>
+        <translation>Software-Shop-Diagnose</translation>
+    </message>
+    <message>
+        <location filename="../home_page_widget.cpp" line="210"/>
+        <source>Resolve issues encountered in software stores</source>
+        <translation>Probleme mit dem Software-Shop beheben</translation>
+    </message>
+    <message>
+        <location filename="../home_page_widget.cpp" line="215"/>
+        <source>System disk availability detection</source>
+        <translation>Systemfestplatten-Verfügbarkeitsprüfung</translation>
+    </message>
+    <message>
+        <location filename="../home_page_widget.cpp" line="216"/>
+        <source>Check the health status of the disk</source>
+        <translation>Festplattengesundheit überprüfen</translation>
+    </message>
+    <message>
+        <location filename="../home_page_widget.cpp" line="221"/>
+        <source>Wired network detection</source>
+        <translation>Überprüfung des Kabelnetzwerks</translation>
+    </message>
+    <message>
+        <location filename="../home_page_widget.cpp" line="222"/>
+        <source>Troubleshooting the issue of not being able to access the internet</source>
+        <translation>Probleme mit der Internetverbindung beheben</translation>
+    </message>
+    <message>
+        <location filename="../home_page_widget.cpp" line="227"/>
+        <source>Wireless network detection</source>
+        <translation>WLAN-Diagnose</translation>
+    </message>
+    <message>
+        <location filename="../home_page_widget.cpp" line="228"/>
+        <source>Troubleshooting wireless network anomalies</source>
+        <translation>Anomalien im WLAN-Netzwerk beheben</translation>
+    </message>
+    <message>
+        <location filename="../home_page_widget.cpp" line="274"/>
+        <source>Internal network detection</source>
+        <translation>Intranet-Diagnoseeinstellungen</translation>
+    </message>
+    <message>
+        <location filename="../home_page_widget.cpp" line="322"/>
+        <source>Preparing...</source>
+        <translation>Wird vorbereitet...</translation>
+    </message>
+</context>
+<context>
+    <name>fault_diagnosis::MainWindow</name>
+    <message>
+        <location filename="../mainwindow.cpp" line="98"/>
+        <location filename="../mainwindow.cpp" line="1855"/>
+        <source>Cancel</source>
+        <translation>Abbrechen</translation>
+    </message>
+    <message>
+        <location filename="../mainwindow.cpp" line="294"/>
+        <source>Detecting...</source>
+        <translation>Wird überprüft...</translation>
+    </message>
+    <message>
+        <location filename="../mainwindow.cpp" line="337"/>
+        <source>No issues fount</source>
+        <translation>Keine Probleme gefunden</translation>
+    </message>
+    <message>
+        <location filename="../mainwindow.cpp" line="326"/>
+        <location filename="../mainwindow.cpp" line="338"/>
+        <source>Rediagnosis</source>
+        <translation>Erneut überprüfen</translation>
+    </message>
+    <message>
+        <location filename="../mainwindow.cpp" line="247"/>
+        <source>Update detection</source>
+        <translation>Aktualisierungsprüfung</translation>
+    </message>
+    <message>
+        <location filename="../mainwindow.cpp" line="250"/>
+        <source>Application detection</source>
+        <translation>Anwendungsdiagnose</translation>
+    </message>
+    <message>
+        <location filename="../mainwindow.cpp" line="306"/>
+        <location filename="../mainwindow.cpp" line="370"/>
+        <source>Canceling in progress...</source>
+        <translation>Wird abgebrochen...</translation>
+    </message>
+    <message>
+        <location filename="../mainwindow.cpp" line="318"/>
+        <source>Detection interruption, no issues found</source>
+        <translation>Erkennung abgebrochen, keine Probleme gefunden</translation>
+    </message>
+    <message>
+        <location filename="../mainwindow.cpp" line="321"/>
+        <source>Detected interruption and found %1 issues</source>
+        <translation>Erkennung abgebrochen, %1 Probleme gefunden</translation>
+    </message>
+    <message>
+        <location filename="../mainwindow.cpp" line="343"/>
+        <source>Found %1 issues</source>
+        <translation>%1 Probleme gefunden</translation>
+    </message>
+    <message>
+        <location filename="../mainwindow.cpp" line="345"/>
+        <source>One click repair</source>
+        <translation>Ein-Klick-Reparatur</translation>
+    </message>
+    <message>
+        <location filename="../mainwindow.cpp" line="359"/>
+        <source>Repairing...</source>
+        <translation>Reparatur läuft...</translation>
+    </message>
+    <message>
+        <location filename="../mainwindow.cpp" line="382"/>
+        <location filename="../mainwindow.cpp" line="404"/>
+        <location filename="../mainwindow.cpp" line="819"/>
+        <location filename="../mainwindow.cpp" line="1525"/>
+        <source>Repair failed</source>
+        <translation>Reparatur fehlgeschlagen</translation>
+    </message>
+    <message>
+        <location filename="../mainwindow.cpp" line="383"/>
+        <source>Repair</source>
+        <translation>Erneut reparieren</translation>
+    </message>
+    <message>
+        <location filename="../mainwindow.cpp" line="387"/>
+        <location filename="../mainwindow.cpp" line="407"/>
+        <location filename="../mainwindow.cpp" line="815"/>
+        <location filename="../mainwindow.cpp" line="1520"/>
+        <source>Repair successful</source>
+        <translation>Reparatur erfolgreich</translation>
+    </message>
+    <message>
+        <location filename="../mainwindow.cpp" line="434"/>
+        <source>%1 items have been tested, no issues found</source>
+        <translation>%1 Elemente überprüft, keine Probleme gefunden</translation>
+    </message>
+    <message>
+        <location filename="../mainwindow.cpp" line="436"/>
+        <source>%1 items have been tested and %2 issues have been found</source>
+        <translation>%1 Elemente überprüft, %2 Probleme gefunden</translation>
+    </message>
+    <message>
+        <location filename="../mainwindow.cpp" line="443"/>
+        <source>I suggest you conduct a complete inspection again</source>
+        <translation>Es wird empfohlen, eine vollständige Überprüfung erneut durchzuführen.</translation>
+    </message>
+    <message>
+        <location filename="../mainwindow.cpp" line="445"/>
+        <source>Please repair and retest</source>
+        <translation>Bitte nach der Reparatur erneut überprüfen</translation>
+    </message>
+    <message>
+        <location filename="../mainwindow.cpp" line="450"/>
+        <source>Please continue to maintain and regularly undergo physical examinations</source>
+        <translation>Bitte machen Sie weiter so und führen Sie regelmäßig Überprüfungen durch</translation>
+    </message>
+    <message>
+        <location filename="../mainwindow.cpp" line="452"/>
+        <source>The scan has been completed. if there are any issues, please handle them as soon as possible</source>
+        <translation>Scan abgeschlossen, bitte beheben Sie die Probleme so schnell wie möglich</translation>
+    </message>
+    <message>
+        <location filename="../mainwindow.cpp" line="456"/>
+        <source>Please wait for the repair to complete</source>
+        <translation>Bitte warten Sie, bis die Reparatur abgeschlossen ist</translation>
+    </message>
+    <message>
+        <location filename="../mainwindow.cpp" line="464"/>
+        <source>There is still %1 issue that has not been successfully fixed. it is recommended to manually fix it</source>
+        <translation>Es gibt noch %1 nicht erfolgreich behobene Probleme. Eine manuelle Reparatur wird empfohlen.</translation>
+    </message>
+    <message>
+        <location filename="../mainwindow.cpp" line="466"/>
+        <source>Regular physical examinations can effectively improve the user experience</source>
+        <translation>Regelmäßige Überprüfungen können das Nutzungserlebnis effektiv verbessern</translation>
+    </message>
+    <message>
+        <location filename="../mainwindow.cpp" line="982"/>
+        <location filename="../mainwindow.cpp" line="1001"/>
+        <source>Backend detection exception</source>
+        <translation>Backend-Überprüfungsanomalie</translation>
+    </message>
+    <message>
+        <location filename="../mainwindow.cpp" line="1265"/>
+        <source>Please select the item that needs to be repaired.</source>
+        <translation>Bitte wählen Sie die zu reparierenden Elemente aus.</translation>
+    </message>
+    <message>
+        <location filename="../mainwindow.cpp" line="1514"/>
+        <source>The issue has been successfully fixed, please try updating again</source>
+        <translation>Das Problem wurde erfolgreich behoben. Bitte versuchen Sie erneut, das Update durchzuführen.</translation>
+    </message>
+    <message>
+        <location filename="../mainwindow.cpp" line="1517"/>
+        <source>The issue has been successfully fixed, please reinstall the software package again</source>
+        <translation>Das Problem wurde erfolgreich behoben. Bitte installieren Sie das Paket erneut.</translation>
+    </message>
+    <message>
+        <location filename="../mainwindow.cpp" line="1854"/>
+        <source>Fault diagnosis is running, are you sure you want to exit?</source>
+        <translation>Die Fehlerdiagnose läuft. Möchten Sie wirklich beenden?</translation>
+    </message>
+    <message>
+        <location filename="../mainwindow.cpp" line="1856"/>
+        <source>Ok</source>
+        <translation>OK</translation>
+    </message>
+    <message>
+        <location filename="../mainwindow.cpp" line="99"/>
+        <source>Return</source>
+        <translation>Zurück</translation>
+    </message>
+</context>
+<context>
+    <name>fault_diagnosis::RepairSpecificErrorWidget</name>
+    <message>
+        <location filename="../repair_specific_error_widget.cpp" line="87"/>
+        <source>Detecting and attempting to fix the issue</source>
+        <translation>Überprüfung und Reparaturversuch laufen</translation>
+    </message>
+    <message>
+        <location filename="../repair_specific_error_widget.cpp" line="96"/>
+        <source>Cancel</source>
+        <translation>Abbrechen</translation>
+    </message>
+    <message>
+        <location filename="../repair_specific_error_widget.cpp" line="220"/>
+        <source>Solution suggestions</source>
+        <translation>Lösungsvorschlag</translation>
+    </message>
+    <message>
+        <location filename="../repair_specific_error_widget.cpp" line="241"/>
+        <location filename="../repair_specific_error_widget.cpp" line="549"/>
+        <source>Report issues</source>
+        <translation>Problem melden</translation>
+    </message>
+    <message>
+        <location filename="../repair_specific_error_widget.cpp" line="262"/>
+        <location filename="../repair_specific_error_widget.cpp" line="393"/>
+        <source>Run</source>
+        <translation>Ausführen</translation>
+    </message>
+    <message>
+        <location filename="../repair_specific_error_widget.cpp" line="264"/>
+        <location filename="../repair_specific_error_widget.cpp" line="395"/>
+        <source>Running...</source>
+        <translation>Läuft...</translation>
+    </message>
+    <message>
+        <location filename="../repair_specific_error_widget.cpp" line="236"/>
+        <source>If the above suggestions still fail to solve the problem, you may need to report the issue to the system maintenance personnel.</source>
+        <translation>Wenn die oben genannten Vorschläge das Problem nicht lösen, sollten Sie das Problem dem Systemadministrator melden.</translation>
+    </message>
+    <message>
+        <location filename="../repair_specific_error_widget.cpp" line="238"/>
+        <source>To attempt to resolve the issue, you may need to report the issue to the system maintenance personnel.</source>
+        <translation>Um zu versuchen, dieses Problem zu beheben, können Sie das Problem an den Systemadministrator melden.</translation>
+    </message>
+    <message>
+        <location filename="../repair_specific_error_widget.cpp" line="260"/>
+        <source>The network is normal but the update cannot be completed, allow the system to run automated scripts to attempt to fix the problem.</source>
+        <translation>Das Netzwerk ist normal, aber das Update kann nicht abgeschlossen werden. Erlauben Sie dem System, ein automatisiertes Skript auszuführen, um das Problem zu beheben.</translation>
+    </message>
+    <message>
+        <location filename="../repair_specific_error_widget.cpp" line="292"/>
+        <source>Please check if the the network connection is normal and update again.</source>
+        <translation>Bitte überprüfen Sie, ob die Netzwerkverbindung normal ist, und versuchen Sie dann erneut, das Update durchzuführen.</translation>
+    </message>
+    <message>
+        <location filename="../repair_specific_error_widget.cpp" line="299"/>
+        <source>Network normal</source>
+        <translation>Netzwerk in Ordnung</translation>
+    </message>
+    <message>
+        <location filename="../repair_specific_error_widget.cpp" line="303"/>
+        <source>Network exception</source>
+        <translation>Netzwerkanomalie</translation>
+    </message>
+    <message>
+        <location filename="../repair_specific_error_widget.cpp" line="326"/>
+        <source>Access</source>
+        <translation>Zugriff</translation>
+    </message>
+    <message>
+        <location filename="../repair_specific_error_widget.cpp" line="328"/>
+        <source>KylinOS official website</source>
+        <translation>KylinOS-Website</translation>
+    </message>
+    <message>
+        <location filename="../repair_specific_error_widget.cpp" line="333"/>
+        <source>view common problems and solutions. For more help, please scan the QR code below: </source>
+        <translation>Häufige Probleme und Lösungen anzeigen. Für weitere Hilfe scannen Sie bitte den folgenden QR-Code: </translation>
+    </message>
+    <message>
+        <location filename="../repair_specific_error_widget.cpp" line="370"/>
+        <source>The system will run automated scripts to try to solve the problem, which will uninstall some software packages or lower the version of some software packages (</source>
+        <translation>Das System wird ein automatisiertes Skript ausführen, um zu versuchen, dieses Problem zu beheben. Dabei werden einige Pakete deinstalliert oder in der Version herabgestuft (</translation>
+    </message>
+    <message>
+        <location filename="../repair_specific_error_widget.cpp" line="373"/>
+        <source>click to view details</source>
+        <translation>Klicken Sie hier für Details</translation>
+    </message>
+    <message>
+        <location filename="../repair_specific_error_widget.cpp" line="375"/>
+        <source>).</source>
+        <translation>).</translation>
+    </message>
+    <message>
+        <location filename="../repair_specific_error_widget.cpp" line="385"/>
+        <source>Is it running?</source>
+        <translation>Ausführen?</translation>
+    </message>
+    <message>
+        <location filename="../repair_specific_error_widget.cpp" line="430"/>
+        <source>Your disk space is insufficient, please click the button on the right to clean it up.</source>
+        <translation>Ihr Speicherplatz ist unzureichend. Bitte klicken Sie auf die Schaltfläche rechts, um ihn zu bereinigen.</translation>
+    </message>
+    <message>
+        <location filename="../repair_specific_error_widget.cpp" line="432"/>
+        <source>Clear</source>
+        <translation>Bereinigen</translation>
+    </message>
+    <message>
+        <location filename="../repair_specific_error_widget.cpp" line="451"/>
+        <source>Return</source>
+        <translation>Zurück</translation>
+    </message>
+    <message>
+        <location filename="../repair_specific_error_widget.cpp" line="492"/>
+        <location filename="../repair_specific_error_widget.cpp" line="509"/>
+        <source>Repair successful</source>
+        <translation>Reparatur erfolgreich</translation>
+    </message>
+    <message>
+        <location filename="../repair_specific_error_widget.cpp" line="495"/>
+        <location filename="../repair_specific_error_widget.cpp" line="512"/>
+        <source>Repair failed</source>
+        <translation>Reparatur fehlgeschlagen</translation>
+    </message>
+    <message>
+        <location filename="../repair_specific_error_widget.cpp" line="525"/>
+        <source>Reporting...</source>
+        <translation>Wird gemeldet...</translation>
+    </message>
+    <message>
+        <location filename="../repair_specific_error_widget.cpp" line="553"/>
+        <source>Successfully reported</source>
+        <translation>Meldung erfolgreich</translation>
+    </message>
+    <message>
+        <location filename="../repair_specific_error_widget.cpp" line="555"/>
+        <source>Report failed</source>
+        <translation>Meldung fehlgeschlagen</translation>
+    </message>
+</context>
+</TS>
diff --git a/plugins/junk_clean/translations/junk-clean_de.ts b/plugins/junk_clean/translations/junk-clean_de.ts
new file mode 100644
index 0000000..ea32587
--- /dev/null
+++ b/plugins/junk_clean/translations/junk-clean_de.ts
@@ -0,0 +1,442 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="de">
+<context>
+    <name>QObject</name>
+    <message>
+        <location filename="../junk_clean.cpp" line="53"/>
+        <source>FileClean</source>
+        <translation>Dateibereinigung</translation>
+    </message>
+    <message>
+        <location filename="../junk_clean.cpp" line="55"/>
+        <source>JunkClean</source>
+        <translation>Datenmüllbereinigung</translation>
+    </message>
+</context>
+<context>
+    <name>junk_clean::BootPartitionCleaner</name>
+    <message>
+        <location filename="../cleaner/boot_partition_cleaner.cpp" line="50"/>
+        <source>boot partition</source>
+        <translation>Boot-Partition</translation>
+    </message>
+    <message>
+        <location filename="../cleaner/boot_partition_cleaner.cpp" line="54"/>
+        <source>Boot partition space cleaning</source>
+        <translation>Speicherplatz der Boot-Partition bereinigen</translation>
+    </message>
+</context>
+<context>
+    <name>junk_clean::CleanUpEntryWidget</name>
+    <message>
+        <location filename="../clean_up_entry_widget.cpp" line="86"/>
+        <source>%1 in total, clean carefully</source>
+        <translation>Insgesamt %1, vorsichtig bereinigen</translation>
+    </message>
+    <message>
+        <location filename="../clean_up_entry_widget.cpp" line="95"/>
+        <source>%1 entries in total, carefully clean up</source>
+        <translation>Insgesamt %1 Einträge, vorsichtig bereinigen</translation>
+    </message>
+    <message>
+        <location filename="../clean_up_entry_widget.cpp" line="98"/>
+        <source>%1 entries</source>
+        <translation>%1 Einträge</translation>
+    </message>
+    <message>
+        <location filename="../clean_up_entry_widget.cpp" line="111"/>
+        <source>Scanning...</source>
+        <translation>Wird gescannt...</translation>
+    </message>
+    <message>
+        <location filename="../clean_up_entry_widget.cpp" line="115"/>
+        <source>Very clean</source>
+        <translation>Sehr sauber</translation>
+    </message>
+</context>
+<context>
+    <name>junk_clean::CleanUpFinishWidget</name>
+    <message>
+        <location filename="../clean_up_finish_widget.cpp" line="77"/>
+        <location filename="../clean_up_finish_widget.cpp" line="173"/>
+        <location filename="../clean_up_finish_widget.cpp" line="180"/>
+        <source>Complete</source>
+        <translation>Fertig</translation>
+    </message>
+    <message>
+        <location filename="../clean_up_finish_widget.cpp" line="87"/>
+        <location filename="../clean_up_finish_widget.cpp" line="172"/>
+        <location filename="../clean_up_finish_widget.cpp" line="181"/>
+        <source>Deep clean</source>
+        <translation>Tiefenbereinigung</translation>
+    </message>
+    <message>
+        <location filename="../clean_up_finish_widget.cpp" line="115"/>
+        <source>%1 of garbage has been cleared</source>
+        <translation>%1 Datenmüll bereinigt</translation>
+    </message>
+    <message>
+        <location filename="../clean_up_finish_widget.cpp" line="176"/>
+        <source>There is only %1 left on the disk, Please perform a deep cleaning</source>
+        <translation>Nur noch %1 auf der Festplatte verfügbar, bitte Tiefenbereinigung durchführen</translation>
+    </message>
+    <message>
+        <location filename="../clean_up_finish_widget.cpp" line="119"/>
+        <source>Cleaning files: %1 entries</source>
+        <translation>Dateien bereinigen: %1</translation>
+    </message>
+    <message>
+        <location filename="../clean_up_finish_widget.cpp" line="123"/>
+        <source>Cleaning time: %1 second</source>
+        <translation>Bereinigungsdauer: %1 Sekunden</translation>
+    </message>
+</context>
+<context>
+    <name>junk_clean::CleanUpGroupWidget</name>
+    <message>
+        <location filename="../clean_up_group_widget.cpp" line="61"/>
+        <source>%1 items of %2 in total, %3 selected</source>
+        <translation>Insgesamt %1 %2, %3 ausgewählt</translation>
+    </message>
+    <message>
+        <location filename="../clean_up_group_widget.cpp" line="67"/>
+        <source>%1 items with %2 traces, %3 selected</source>
+        <translation>Insgesamt %1 Elemente, %2 Spuren, %3 ausgewählt</translation>
+    </message>
+    <message>
+        <location filename="../clean_up_group_widget.cpp" line="91"/>
+        <source>System junk</source>
+        <translation>Systemdatenmüll</translation>
+    </message>
+    <message>
+        <location filename="../clean_up_group_widget.cpp" line="93"/>
+        <source>Internet junk</source>
+        <translation>Internet-Datenmüll</translation>
+    </message>
+    <message>
+        <location filename="../clean_up_group_widget.cpp" line="95"/>
+        <source>Usage traces</source>
+        <translation>Nutzungsspuren</translation>
+    </message>
+    <message>
+        <location filename="../clean_up_group_widget.cpp" line="97"/>
+        <source>Other</source>
+        <translation>Sonstiger Datenmüll</translation>
+    </message>
+</context>
+<context>
+    <name>junk_clean::CommandLineTraceCleaner</name>
+    <message>
+        <location filename="../cleaner/command_line_trace_cleaner.cpp" line="40"/>
+        <source>Command line history</source>
+        <translation>Befehlszeilenverlauf</translation>
+    </message>
+    <message>
+        <location filename="../cleaner/command_line_trace_cleaner.cpp" line="44"/>
+        <source>Historical traces generated by using the command line</source>
+        <translation>Über die Befehlszeile generierter Verlauf</translation>
+    </message>
+</context>
+<context>
+    <name>junk_clean::EnterpriseWechatCleaner</name>
+    <message>
+        <location filename="../cleaner/enterprise_wechat_cleaner.cpp" line="28"/>
+        <source>Enterprise Wechat</source>
+        <translation>WeCom</translation>
+    </message>
+    <message>
+        <location filename="../cleaner/enterprise_wechat_cleaner.cpp" line="32"/>
+        <source>Clean up files received on enterprise wechat</source>
+        <translation>Von WeCom empfangene Dateien bereinigen</translation>
+    </message>
+</context>
+<context>
+    <name>junk_clean::FileTraceCleaner</name>
+    <message>
+        <location filename="../cleaner/file_trace_cleaner.cpp" line="39"/>
+        <source>File recording</source>
+        <translation>Dateiprotokolle</translation>
+    </message>
+    <message>
+        <location filename="../cleaner/file_trace_cleaner.cpp" line="43"/>
+        <source>Recently opened file records</source>
+        <translation>Verlauf der zuletzt geöffneten Dateien</translation>
+    </message>
+</context>
+<context>
+    <name>junk_clean::Homepage</name>
+    <message>
+        <location filename="../homepage.cpp" line="32"/>
+        <source>One click cleaning of computer junk</source>
+        <translation>PC-Datenmüll Ein-Klick-Bereinigung</translation>
+    </message>
+    <message>
+        <location filename="../homepage.cpp" line="34"/>
+        <source>Regular cleaning makes your computer light, fast, and safe</source>
+        <translation>Regelmäßiges Bereinigen hält Ihren PC schnell und sicher</translation>
+    </message>
+    <message>
+        <location filename="../homepage.cpp" line="52"/>
+        <source>System garbage</source>
+        <translation>Systemdatenmüll</translation>
+    </message>
+    <message>
+        <location filename="../homepage.cpp" line="54"/>
+        <source>Clean up the garbage generated during the daily use of the system</source>
+        <translation>Im täglichen Systembetrieb entstandenen Datenmüll bereinigen</translation>
+    </message>
+    <message>
+        <location filename="../homepage.cpp" line="71"/>
+        <source>Interface garbage</source>
+        <translation>Internet-Datenmüll</translation>
+    </message>
+    <message>
+        <location filename="../homepage.cpp" line="73"/>
+        <source>Cleaning browser generated cache files and login information</source>
+        <translation>Vom Browser generierte Cache-Dateien und Anmeldeinformationen bereinigen</translation>
+    </message>
+    <message>
+        <location filename="../homepage.cpp" line="90"/>
+        <source>Usage traces</source>
+        <translation>Nutzungsspuren</translation>
+    </message>
+    <message>
+        <location filename="../homepage.cpp" line="92"/>
+        <source>Clean up computer usage traces and protect privacy and security</source>
+        <translation>PC-Nutzungsspuren bereinigen, Privatsphäre schützen</translation>
+    </message>
+    <message>
+        <location filename="../homepage.cpp" line="110"/>
+        <source>Start scan</source>
+        <translation>Scan starten</translation>
+    </message>
+</context>
+<context>
+    <name>junk_clean::InstallationPackageCleaner</name>
+    <message>
+        <location filename="../cleaner/installation_package_cleaner.cpp" line="55"/>
+        <source>Installation package cache</source>
+        <translation>Installationspaket-Cache</translation>
+    </message>
+    <message>
+        <location filename="../cleaner/installation_package_cleaner.cpp" line="59"/>
+        <source>Garbage files generated during software installation</source>
+        <translation>Bei der Softwareinstallation entstandene überflüssige Dateien</translation>
+    </message>
+</context>
+<context>
+    <name>junk_clean::JunkEntryWidget</name>
+    <message>
+        <location filename="../junk_entry_widget.cpp" line="28"/>
+        <source>0 entries</source>
+        <translation>0 Einträge</translation>
+    </message>
+    <message>
+        <location filename="../junk_entry_widget.cpp" line="30"/>
+        <source>0 B</source>
+        <translation>0 B</translation>
+    </message>
+    <message>
+        <location filename="../junk_entry_widget.cpp" line="75"/>
+        <source>%1 entries</source>
+        <translation>%1 Einträge</translation>
+    </message>
+</context>
+<context>
+    <name>junk_clean::LogCleaner</name>
+    <message>
+        <location filename="../cleaner/log_cleaner.cpp" line="68"/>
+        <source>Log file</source>
+        <translation>Protokolldateien</translation>
+    </message>
+    <message>
+        <location filename="../cleaner/log_cleaner.cpp" line="72"/>
+        <source>Logs generated by software operation</source>
+        <translation>Während des Softwarebetriebs erstellte Protokolle</translation>
+    </message>
+</context>
+<context>
+    <name>junk_clean::MainWindow</name>
+    <message>
+        <location filename="../mainwindow.cpp" line="66"/>
+        <source>Return</source>
+        <translation>Zurück</translation>
+    </message>
+    <message>
+        <location filename="../mainwindow.cpp" line="68"/>
+        <source>Clean</source>
+        <translation>Ein-Klick-Bereinigung</translation>
+    </message>
+    <message>
+        <location filename="../mainwindow.cpp" line="204"/>
+        <source>Scanning in progress...</source>
+        <translation>Wird gescannt...</translation>
+    </message>
+    <message>
+        <location filename="../mainwindow.cpp" line="208"/>
+        <source>Cancel scan</source>
+        <translation>Scan abbrechen</translation>
+    </message>
+    <message>
+        <location filename="../mainwindow.cpp" line="214"/>
+        <location filename="../mainwindow.cpp" line="222"/>
+        <source>Canceling in progress</source>
+        <translation>Wird abgebrochen</translation>
+    </message>
+    <message>
+        <location filename="../mainwindow.cpp" line="252"/>
+        <source>No cleanable files found</source>
+        <translation>Keine zu bereinigenden Dateien gefunden</translation>
+    </message>
+    <message>
+        <location filename="../mainwindow.cpp" line="254"/>
+        <source>Found %1 of cleanable files in total</source>
+        <translation>Insgesamt %1 bereinigbare Dateien gefunden</translation>
+    </message>
+    <message>
+        <location filename="../mainwindow.cpp" line="256"/>
+        <source>Selected %1 of garbage and %2 usage traces</source>
+        <translation>%1 Datenmüll und %2 Nutzungsspuren ausgewählt</translation>
+    </message>
+    <message>
+        <location filename="../mainwindow.cpp" line="277"/>
+        <source>%1 garbage cleaning in progress...</source>
+        <translation>%1 Datenmüll wird bereinigt...</translation>
+    </message>
+    <message>
+        <location filename="../mainwindow.cpp" line="279"/>
+        <source>Cleaning up %1 traces...</source>
+        <translation>%1 Nutzungsspuren werden bereinigt...</translation>
+    </message>
+    <message>
+        <location filename="../mainwindow.cpp" line="287"/>
+        <source>Canecl cleanup</source>
+        <translation>Bereinigung abbrechen</translation>
+    </message>
+    <message>
+        <location filename="../mainwindow.cpp" line="336"/>
+        <source>Scanning for garbage...</source>
+        <translation>Datenmüll wird gescannt...</translation>
+    </message>
+    <message>
+        <location filename="../mainwindow.cpp" line="375"/>
+        <source>Scanning: %1</source>
+        <translation>Wird gescannt: %1</translation>
+    </message>
+    <message>
+        <location filename="../mainwindow.cpp" line="780"/>
+        <source>Cleaning up garbage...</source>
+        <translation>Datenmüll wird bereinigt...</translation>
+    </message>
+    <message>
+        <location filename="../mainwindow.cpp" line="839"/>
+        <source>Cleaning up: %1</source>
+        <translation>Wird bereinigt: %1</translation>
+    </message>
+</context>
+<context>
+    <name>junk_clean::QQCleaner</name>
+    <message>
+        <location filename="../cleaner/qq_cleaner.cpp" line="75"/>
+        <source>QQ</source>
+        <translation>QQ</translation>
+    </message>
+    <message>
+        <location filename="../cleaner/qq_cleaner.cpp" line="79"/>
+        <source>Temporary data generated using QQ</source>
+        <translation>Durch die Nutzung von QQ erzeugte temporäre Daten</translation>
+    </message>
+</context>
+<context>
+    <name>junk_clean::QaxBrowserCleaner</name>
+    <message>
+        <location filename="../cleaner/qax_browser_cleaner.cpp" line="39"/>
+        <source>QaxBrowser</source>
+        <translation>Qi-Anxin-Browser</translation>
+    </message>
+    <message>
+        <location filename="../cleaner/qax_browser_cleaner.cpp" line="43"/>
+        <source>Some web pages or resources cached by the browser when loading pages</source>
+        <translation>Vom Browser beim Laden von Seiten zwischengespeicherte Webseiten oder Ressourcen</translation>
+    </message>
+</context>
+<context>
+    <name>junk_clean::QaxBrowserCookiesCleaner</name>
+    <message>
+        <location filename="../cleaner/qax_browser_cookies_cleaner.cpp" line="39"/>
+        <source>Cookies information</source>
+        <translation>Cookie-Informationen</translation>
+    </message>
+    <message>
+        <location filename="../cleaner/qax_browser_cookies_cleaner.cpp" line="43"/>
+        <source>Browser saved login information</source>
+        <translation>Vom Browser gespeicherte Anmeldeinformationen</translation>
+    </message>
+</context>
+<context>
+    <name>junk_clean::ResidualCleaner</name>
+    <message>
+        <location filename="../cleaner/residual_cleaner.cpp" line="53"/>
+        <source>Uninstall residual</source>
+        <translation>Deinstallationsreste</translation>
+    </message>
+    <message>
+        <location filename="../cleaner/residual_cleaner.cpp" line="57"/>
+        <source>Residual configuration files and cache after software uninstallation</source>
+        <translation>Verbleibende Konfigurationsdateien und Caches nach der Deinstallation von Software</translation>
+    </message>
+</context>
+<context>
+    <name>junk_clean::SystemMemoryCleaner</name>
+    <message>
+        <location filename="../cleaner/system_memory_cleaner.cpp" line="33"/>
+        <source>System cache</source>
+        <translation>Systemcache</translation>
+    </message>
+    <message>
+        <location filename="../cleaner/system_memory_cleaner.cpp" line="37"/>
+        <source>Release system memory</source>
+        <translation>Systemspeicher freigeben</translation>
+    </message>
+</context>
+<context>
+    <name>junk_clean::ThumbnailCleaner</name>
+    <message>
+        <location filename="../cleaner/thumbnail_cleaner.cpp" line="40"/>
+        <source>Thumbnail cache</source>
+        <translation>Miniaturansichten-Cache</translation>
+    </message>
+    <message>
+        <location filename="../cleaner/thumbnail_cleaner.cpp" line="44"/>
+        <source>After cleaning, viewing thumbnails will slow dowm. Please clean as needed</source>
+        <translation>Nach der Bereinigung wird das Anzeigen von Miniaturansichten langsamer, bitte nach Bedarf bereinigen</translation>
+    </message>
+</context>
+<context>
+    <name>junk_clean::TrashCleaner</name>
+    <message>
+        <location filename="../cleaner/trash_cleaner.cpp" line="26"/>
+        <source>Trash</source>
+        <translation>Papierkorb</translation>
+    </message>
+    <message>
+        <location filename="../cleaner/trash_cleaner.cpp" line="30"/>
+        <source>Before cleaning, please confirm that all files in the recycle bin are removable</source>
+        <translation>Bitte stellen Sie vor der Bereinigung sicher, dass alle Dateien im Papierkorb gelöscht werden können</translation>
+    </message>
+</context>
+<context>
+    <name>junk_clean::WechatCleaner</name>
+    <message>
+        <location filename="../cleaner/wechat_cleaner.cpp" line="153"/>
+        <source>Wechat</source>
+        <translation>WeChat</translation>
+    </message>
+    <message>
+        <location filename="../cleaner/wechat_cleaner.cpp" line="157"/>
+        <source>Clean up files received on wechat</source>
+        <translation>Von WeChat empfangene Dateien bereinigen</translation>
+    </message>
+</context>
+</TS>
diff --git a/treasure-chest/file-smash/translations/kylin-file-crush_ko_KP.ts b/treasure-chest/file-smash/translations/kylin-file-crush_ko_KP.ts
index 1c3e3eb..6e3270f 100644
--- a/treasure-chest/file-smash/translations/kylin-file-crush_ko_KP.ts
+++ b/treasure-chest/file-smash/translations/kylin-file-crush_ko_KP.ts
@@ -6,7 +6,7 @@
     <message>
         <location filename="../alertdialog.ui" line="14"/>
         <source>Form</source>
-        <translation type="unfinished"></translation>
+        <translation>형성</translation>
     </message>
 </context>
 <context>
@@ -14,7 +14,7 @@
     <message>
         <location filename="../shreddialog.cpp" line="977"/>
         <source>Click the “+” button or directly drag and drop files here</source>
-        <translation type="unfinished"></translation>
+        <translation>“+” 버튼을 클릭하거나 파일을 직접 여기로 드래그하십시오</translation>
     </message>
 </context>
 <context>
@@ -41,12 +41,12 @@
     <message>
         <location filename="../kalertdialog.cpp" line="47"/>
         <source>The file you selected is a directory or does not have permissions, cannot be shredded!</source>
-        <translation type="unfinished"></translation>
+        <translation>선택한 파일이 디렉토리이거나 권한이 없어 분쇄할 수 없습니다!</translation>
     </message>
     <message>
         <location filename="../kalertdialog.cpp" line="50"/>
         <source>sure</source>
-        <translation type="unfinished"></translation>
+        <translation>삭제</translation>
     </message>
 </context>
 <context>
@@ -54,22 +54,22 @@
     <message>
         <location filename="../ksuredialog.cpp" line="48"/>
         <source>Are you sure to start crushing?</source>
-        <translation type="unfinished"></translation>
+        <translation>분쇄를 시작하시겠습니까?</translation>
     </message>
     <message>
         <location filename="../ksuredialog.cpp" line="54"/>
         <source>Crushed files cannot be recovered!</source>
-        <translation type="unfinished"></translation>
+        <translation>분쇄된 파일은 복구할 수 없습니다!</translation>
     </message>
     <message>
         <location filename="../ksuredialog.cpp" line="62"/>
         <source>sure</source>
-        <translation type="unfinished"></translation>
+        <translation>삭제</translation>
     </message>
     <message>
         <location filename="../ksuredialog.cpp" line="70"/>
         <source>cancle</source>
-        <translation type="unfinished"></translation>
+        <translation>취소</translation>
     </message>
 </context>
 <context>
@@ -82,63 +82,63 @@
         <location filename="../shreddialog.cpp" line="38"/>
         <location filename="../shreddialog.cpp" line="39"/>
         <source>File Crush</source>
-        <translation type="unfinished"></translation>
+        <translation>파일분쇄기</translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="65"/>
         <location filename="../shreddialog.cpp" line="815"/>
         <source>File shredding, please wait a moment</source>
-        <translation type="unfinished"></translation>
+        <translation>파일 분쇄 중입니다. 잠시 기다리십시오</translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="113"/>
         <source>No file selected to be shredded</source>
-        <translation type="unfinished"></translation>
+        <translation>분쇄할 파일이 선택되지 않았습니다</translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="131"/>
         <source>select all</source>
-        <translation type="unfinished"></translation>
+        <translation>전체선택</translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="227"/>
         <source>File Name</source>
-        <translation type="unfinished"></translation>
+        <translation>파일 이름</translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="227"/>
         <source>File Path</source>
-        <translation type="unfinished"></translation>
+        <translation>파일 전송 경로</translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="394"/>
         <source>Loading file information... </source>
-        <translation type="unfinished"></translation>
+        <translation>파일정보 불러오는 중... </translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="577"/>
         <source>Completely crush files or folders that cannot be deleted</source>
-        <translation type="unfinished"></translation>
+        <translation>삭제할 수 없는 파일이나 폴더를 완전히 분쇄합니다</translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="578"/>
         <source>Shred File</source>
-        <translation type="unfinished"></translation>
+        <translation>분쇄</translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="579"/>
         <source>Deselect</source>
-        <translation type="unfinished"></translation>
+        <translation>제거</translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="580"/>
         <source>Add File</source>
-        <translation type="unfinished"></translation>
+        <translation>추가</translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="581"/>
         <source>Note:File shredding cannot be cancelled, please exercise caution!</source>
-        <translation type="unfinished"></translation>
+        <translation>*파일 파기 과정은 취소할 수 없으니 신중하게 작업하십시오!</translation>
     </message>
     <message>
         <source>Select file</source>
@@ -147,24 +147,24 @@
     <message>
         <location filename="../shreddialog.cpp" line="835"/>
         <source> </source>
-        <translation type="unfinished"></translation>
+        <translation> </translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="835"/>
         <source> File shredding failed</source>
-        <translation type="unfinished"></translation>
+        <translation> 파일 분쇄에 실패했습니다</translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="837"/>
         <location filename="../shreddialog.cpp" line="840"/>
         <source>cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>취소</translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="838"/>
         <location filename="../shreddialog.cpp" line="843"/>
         <source>view info</source>
-        <translation type="unfinished"></translation>
+        <translation>상세정보</translation>
     </message>
     <message>
         <source>All Files(*)</source>
@@ -173,12 +173,12 @@
     <message>
         <location filename="../shreddialog.cpp" line="781"/>
         <source>File shredding completed</source>
-        <translation type="unfinished"></translation>
+        <translation>파일 분쇄가 완료되었습니다</translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="834"/>
         <source>error</source>
-        <translation type="unfinished"></translation>
+        <translation>오류</translation>
     </message>
     <message>
         <source>File shredding failed</source>
@@ -187,12 +187,12 @@
     <message>
         <location filename="../shreddialog.cpp" line="858"/>
         <source>Please select the file to shred!</source>
-        <translation type="unfinished"></translation>
+        <translation>분쇄할 파일을 선택하십시오!</translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="886"/>
         <source>Please select the file to cacel!</source>
-        <translation type="unfinished"></translation>
+        <translation>제거할 파일을 선택하십시오!</translation>
     </message>
 </context>
 <context>
@@ -200,17 +200,17 @@
     <message>
         <location filename="../shredfailinfodialog.cpp" line="22"/>
         <source>cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>취소</translation>
     </message>
     <message>
         <location filename="../shredfailinfodialog.cpp" line="100"/>
         <source>File Name</source>
-        <translation type="unfinished"></translation>
+        <translation>파일 이름</translation>
     </message>
     <message>
         <location filename="../shredfailinfodialog.cpp" line="100"/>
         <source>File Path</source>
-        <translation type="unfinished"></translation>
+        <translation>파일 전송 경로</translation>
     </message>
 </context>
 <context>
@@ -218,12 +218,12 @@
     <message>
         <location filename="../shredmanager.cpp" line="49"/>
         <source>Shred Manager</source>
-        <translation type="unfinished"></translation>
+        <translation>파일분쇄기</translation>
     </message>
     <message>
         <location filename="../shredmanager.cpp" line="54"/>
         <source>Delete files makes it unable to recover</source>
-        <translation type="unfinished"></translation>
+        <translation>파일을 분쇄하면 복구할 수 없습니다</translation>
     </message>
 </context>
 </TS>
